### PR TITLE
feat(lb): add human marshaller for frontend

### PR DESCRIPTION
**Pull request by remyleone**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request.
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000

Release note for [CHANGELOG](https://github.com/scaleway/scaleway-cli/blob/v2/CHANGELOG.md):

```release-note

```

### CODE REVIEW

File: internal/namespaces/lb/v1/custom_frontend.go [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package lb
+
+import "github.com/scaleway/scaleway-cli/internal/core"
+
+func frontendGetBuilder(c *core.Command) *core.Command {
+	c.View = &core.View{
+		Sections: []*core.ViewSection{
+			{
+				FieldName: "LB",
```
Comment (jerome-quere) [inline internal/namespaces/lb/v1/custom_frontend.go:9]: I know Lb is included in the frontend response but do we want to print it ?

File: internal/namespaces/lb/v1/testdata/test-get-frontend-simple.golden [hunk 2]
```diff
@@ -0,0 +1,10 @@
+🎲🎲🎲 EXIT CODE: 1 🎲🎲🎲
+🟥🟥🟥 STDERR️️ 🟥🟥🟥️
+Not found
```
Comment (jerome-quere) [inline internal/namespaces/lb/v1/testdata/test-get-frontend-simple.golden:3]: not normal

File: internal/namespaces/lb/v1/custom_frontend.go (hunk 1 above)
Comment (remyleone) [reply internal/namespaces/lb/v1/custom_frontend.go:9]: #1286 offers the possibility to hide fields. We can open another PR if needed.

File: internal/namespaces/lb/v1/testdata/test-get-frontend-simple.golden (hunk 2 above)
Comment (kindermoumoute) [reply internal/namespaces/lb/v1/testdata/test-get-frontend-simple.golden:3]: This is probable because LB does not return a standard error yet

**Review by jerome-quere — APPROVED** [review]
LGTM

**Pull request merged**